mealie/services/scheduler/scheduler_service.py

```diff
@@ -18,18 +21,43 @@ class SchedulerService:
     @staticmethod
     async def start():
         await run_minutely()
-        await run_daily()
         await run_hourly()
 
+        # Wait to trigger our daily run until our given "daily time", so having asyncio handle it.
+        asyncio.create_task(schedule_daily())
+
+
+async def schedule_daily():
+    now = datetime.now()
+    daily_schedule_time = get_app_settings().DAILY_SCHEDULE_TIME
+    logger.debug(
+        "Current time is %s and DAILY_SCHEDULE_TIME is %s",
+        str(now),
+        daily_schedule_time,
+    )
+    hour_target = int(daily_schedule_time.split(":")[0])
```

Comment:
Can we add a try/catch on the `daily_schedule_time` parsing? Maybe bump it to its own method? That way this doesn't fail if the string is malformed. We could probably default to midnight and log the exception:

```python
try:
    hour_target, minute_target = _parse_daily_schedule_time(daily_schedule_time)
except Exception as e:
    logger.exception(f"Unable to parse {daily_schedule_time=})
    hour_target = minute_target = 0
```

Reply:
Updated. Test:

```
mealie  | DEBUG    2024-05-26T13:20:41 - Current time is 2024-05-26 13:20:41.016770 and DAILY_SCHEDULE_TIME is ThisIsBadData
mealie  | ERROR    2024-05-26T13:20:41 - Unable to parse daily_schedule_time='ThisIsBadData'
mealie  | Traceback (most recent call last):
mealie  |   File "/app/mealie/services/scheduler/scheduler_service.py", line 39, in schedule_daily
mealie  |     hour_target, minute_target = _parse_daily_schedule_time(daily_schedule_time)
mealie  |   File "/app/mealie/services/scheduler/scheduler_service.py", line 60, in _parse_daily_schedule_time
mealie  |     hour_target = int(time.split(":")[0])
mealie  | ValueError: invalid literal for int() with base 10: 'ThisIsBadData'
mealie  | ERROR    2024-05-26T13:20:41 - Unable to parse daily_schedule_time='ThisIsBadData'
mealie  | Traceback (most recent call last):
mealie  |   File "/app/mealie/services/scheduler/scheduler_service.py", line 39, in schedule_daily
mealie  |     hour_target, minute_target = _parse_daily_schedule_time(daily_schedule_time)
mealie  |   File "/app/mealie/services/scheduler/scheduler_service.py", line 60, in _parse_daily_schedule_time
mealie  |     hour_target = int(time.split(":")[0])
mealie  | ValueError: invalid literal for int() with base 10: 'ThisIsBadData'
mealie  | DEBUG    2024-05-26T13:20:41 - Hours until 10 and minutes until 25
mealie  | INFO     2024-05-26T13:20:41 - Daily tasks scheduled for 2024-05-26 23:45:00
mealie  | INFO     2024-05-26T13:20:41 - Application startup complete.
mealie  | INFO     2024-05-26T13:20:41 - Uvicorn running on http://0.0.0.0:9000 (Press CTRL+C to quit)
```